packages/utils/dropdown/ResourceDropdown.tsx

```diff
@@ -218,6 +220,7 @@ const ResourceDropdown: ResourceDropdown = <T extends unknown>({
           onToggle={loaded && !loadError ? onToggle : () => {}}
           toggleIndicator={CaretDownIcon}
           data-test={`${dataTest}-toggle`}
+          isDisabled={disabled}
```

Comment:
if we add `isDisabled` here, this component will unnecessarily still call
  `const [resources, loaded, loadError] =
    useK8sWatchResource<T[]>(watchResource);`
   first condition is to check if `isDisabled` is true return disabled dropdown...

Reply:
then current functioning of the component...